### PR TITLE
BJEJ-30 add btn on ninja page to return to identification page

### PR DESCRIPTION
**Pull request by LarryMatte**
screen shot 2015-05-07 at 4 46 58 pm

### CODE REVIEW

File: src/main/java/com/arcbees/bourseje/client/application/alreadyvoted/AlreadyVotedView.ui.xml [hunk 1]
```diff
@@ -9,42 +9,45 @@
...
+                                c0,0.634,0.514,1.149,1.149,1.149c0.634,0,1.149-0.515,1.149-1.149c0-0.214-0.062-0.411-0.164-0.583L23.481,24.95z"/>
+                            <path fill="#010101" d="M31.037,24.103l3.344-0.503l-0.628-1.712l-8.35,3.061l3.878-0.582c-0.103,0.172-0.163,0.369-0.163,0.582
+                                c0,0.635,0.514,1.15,1.148,1.15s1.149-0.515,1.149-1.15C31.415,24.613,31.268,24.312,31.037,24.103"/>
+                        </svg>
+                    </div>
+                    <div class="{res.styles.text_main}">
+                        <h2>Code déjà utilisé</h2>
+                        <p>Pas la peine d'essayer <span>de se déguiser en ninja, on t'a reconnu!</span></p>
+                        <a href="#/identification" class="{res.styles.btn} {res.styles.btn_wide}">
+                            Entrer un nouveau code
```
Comment (christiangoudreau) [inline src/main/java/com/arcbees/bourseje/client/application/alreadyvoted/AlreadyVotedView.ui.xml]: Entrez?

**Comment by christiangoudreau** [comment]
LGTM

File: src/main/java/com/arcbees/bourseje/client/application/alreadyvoted/AlreadyVotedView.ui.xml (hunk 1 above)
Comment (jclariviere) [inline src/main/java/com/arcbees/bourseje/client/application/alreadyvoted/AlreadyVotedView.ui.xml]: "Entrer" is better, describes what the button does.

**Comment by jclariviere** [comment]
LGTM

**Pull request merged**